src/Umbraco.Core/Constants-Configuration.cs

```diff
@@ -48,6 +48,7 @@ public static class Configuration
         public const string ConfigRequestHandler = ConfigPrefix + "RequestHandler";
         public const string ConfigRuntime = ConfigPrefix + "Runtime";
         public const string ConfigRuntimeMode = ConfigRuntime + ":Mode";
+        [Obsolete("Runtime minification is no longer supported. Will be removed entirely in V16.")]
         public const string ConfigRuntimeMinification = ConfigPrefix + "RuntimeMinification";
         public const string ConfigRuntimeMinificationVersion = ConfigRuntimeMinification + ":Version";
```

Comment:
We can obsolete this one as well
```suggestion
        [Obsolete("Runtime minification is no longer supported. Will be removed entirely in V16.")]
        public const string ConfigRuntimeMinificationVersion = ConfigRuntimeMinification + ":Version";
```

Reply:
Ahh yes, didn't even notice that, thanks 🙌